app/modules/node.c

```diff
@@ -58,6 +58,7 @@ static int node_setonerror( lua_State* L ) {
 }
 
 // Lua: startupcommand(string)
+// The lua.startup({command="string"}) should be used instead
```

Comment:
Add a deprecation warning?

Reply:
I add a message as a comment and also updated the documentation